_schema.graphql

```diff
@@ -4489,9 +4489,10 @@ type Metaphysics {
   queryTracing: Boolean!
   heapDumps: Boolean!
   stitching: Boolean!
-  stitchingGravity: Boolean!
   stitchingConvection: Boolean!
   stitchingExchange: Boolean!
+  stitchingGravity: Boolean!
+  stitchingKaws: Boolean!
 }
```

Comment:
Expands the GraphQL MP config schema to note that kaws is also happening

Reply:
Oh didn’t know about this piece of introspection 👍 (What’s `environment: Boolean!`?)